cmd/ci-images-mirror/main.go

```diff
@@ -102,7 +106,11 @@ func (o *options) loadConfig(caller string) error {
 		return nil
 	}
 	logrus.WithField("caller", caller).Info("Loading config ...")
-	c, err := quayiociimagesdistributor.LoadConfig(o.configFile)
+	bytes, err := gzip.ReadFileMaybeGZIP(o.configFile)
```

Comment:
NIT: I would define `quayiociimagesdistributor.LoadConfigFromFile(file string)` and `LoadConfig(bytes)` might become private.